{{cookiecutter.project_name}}/.github/workflows/pypi-publish.yaml

```diff
@@ -18,18 +18,17 @@ jobs:
         python-version: 3.9
 
     - name: Install Poetry
-      uses: snok/install-poetry@v1.1.6
-      with:
-        virtualenvs-create: true
-        virtualenvs-in-project: true
+      run: pip install poetry poetry-dynamic-versioning
+      # uses: snok/install-poetry@v1.1.6
+      # with:
+      #   virtualenvs-create: true
+      #   virtualenvs-in-project: true
 
     - name: Install dependencies
       run: poetry install --no-interaction
 
```

Comment:
An install is not required to build. Remove section?

Reply:
you're right. I have commented this out just in case.